src/hardware/hardware-add-on-data-types.md

```diff
@@ -3,696 +3,4 @@ layout: page
-| All run hours | 34852 | s | 4 | 360 | 0 |
-| Park idle hours | 34853 | s | 4 | 360 | 0 |
-| Park run hours | 34854 | s | 4 | 360 | 0 |
-| In drive count | 34855 | 0 | 4 | 1 | 0 |
-| Accelerator override count | 34856 | 0 | 4 | 1 | 0 |
-| Power only hours | 34857 | s | 4 | 360 | 0 |
-| Power only count | 34858 | 0 | 4 | 1 | 0 |
-| IMS cooling hours | 34859 | s | 4 | 360 | 0 |
-| IMS cooling count | 34860 | 0 | 4 | 1 | 0 |
-| IMS heating hours | 34861 | s | 4 | 360 | 0 |
-| IMS heating count | 34862 | 0 | 4 | 1 | 0 |
-| Battery 1 charging hours | 34863 | s | 4 | 360 | 0 |
-| Battery 1 charging count | 34864 | 0 | 4 | 1 | 0 |
-| Battery 2 charging hours | 34865 | s | 4 | 360 | 0 |
-| Battery 2 charging count | 34866 | 0 | 4 | 1 | 0 |
-| Brake override hours | 34867 | s | 4 | 360 | 0 |
-| Brake override count | 34868 | 0 | 4 | 1 | 0 |
-| Idle cooling hours | 34869 | s | 4 | 360 | 0 |
-| Idle cooling count | 34870 | 0 | 4 | 1 | 0 |
-| Idle heating hours | 34871 | s | 4 | 360 | 0 |
-| Idle heating count | 34872 | 0 | 4 | 1 | 0 |
-| Input override hours | 34873 | s | 4 | 360 | 0 |
-| Input override count | 34874 | 0 | 4 | 1 | 0 |
-| Aux 1 hours | 34875 | s | 4 | 360 | 0 |
-| Aux 1 count | 34876 | 0 | 4 | 1 | 0 |
-| Aux 2 hours | 34877 | s | 4 | 360 | 0 |
-| Aux 2 count | 34878 | 0 | 4 | 1 | 0 |
-| ePTO active hours | 34879 | s | 4 | 360 | 0 |
-| ePTO active count | 34880 | 0 | 4 | 1 | 0 |
-| Battery 1 minimum voltage | 34881 | V | 4 | 0.1 | 0 |
-| Battery 2 minimum voltage | 34882 | V | 4 | 0.1 | 0 |
-| Shore power hours | 34883 | s | 4 | 360 | 0 |
-| Shore power count | 34884 | 0 | 4 | 1 | 0 |
-| A/C error code 1 | 34885 | 0 | 4 | 1 | 0 |
-| A/C error code 2 | 34886 | 0 | 4 | 1 | 0 |
-| A/C error code 3 | 34887 | 0 | 4 | 1 | 0 |
-| Remote disconnect 1 | 34888 | 0 | 4 | 1 | 0 |
-| Remote disconnect 2 | 34889 | 0 | 4 | 1 | 0 |
-| Remote disconnect 3 | 34890 | 0 | 4 | 1 | 0 |
-| Start error 1 | 34891 | 0 | 4 | 1 | 0 |
-| Start error 2 | 34892 | 0 | 4 | 1 | 0 |
-| Start error 3 | 34893 | 0 | 4 | 1 | 0 |
-| Generic external device serial number | 34894 | 0 | 4 | 1 | 0 |
-| Generic fuel added | 34895 | L | 2 | 0.01 | 0 |
-| Generic fuel siphoned | 34896 | L | 2 | 0.01 | 0 |
-| Generic external device hardware version | 34897 | 0 | 4 | 0.1 | 0 |
-| Generic external device firmware version | 34898 | 0 | 4 | 0.01 | 0 |
-| Generic external device temperature | 34899 | C | 4 | 0.1 | 0 |
-| Generic external device fuel level | 34900 | % | 4 | 0.1 | 0 |
-| Generic external device AC current 1 (RMS) | 34901 | A | 4 | 0.1 | 0 |
-| Generic external device AC current 2 (RMS) | 34902 | A | 4 | 0.1 | 0 |
-| Generic external device status | 34903 | 0 | 4 | 1 | 0 |
-| Welder voltage | 34904 | V | 4 | 0.001 | 0 |
-| Welder current | 34905 | A | 4 | 0.001 | 0 |
+This page has been deprecated. The information formerly provided on this page can now be found [Here].(https://docs.google.com/spreadsheets/d/1sy7IOhWmFoo40_I-ruOJO8bVTMLXqHa11d0oJtaoIcE/edit#gid=1405610761s).
```

Comment:
Could you change this to https://docs.google.com/spreadsheets/d/1sy7IOhWmFoo40_I-ruOJO8bVTMLXqHa11d0oJtaoIcE/edit#gid=1133172080 ? We'd like to land people on the README page